src/components/Message/__tests__/MessageCommerce.test.js

```diff
@@ -28,15 +34,13 @@ async function renderMessageCommerce(
   const channel = generateChannel({ getConfig: () => channelConfig });
   const client = await getTestClientWithUser(alice);
   return render(
-    <Message
-      t={(key) => key}
-      channel={channel}
-      client={client}
-      message={message}
-      typing={false}
-      Message={MessageCommerce}
-      {...props}
-    />,
+    <ChannelContext.Provider
+      value={{ channel, client, openThread: openThreadMock }}
+    >
+      <TranslationContext.Provider value={{ t: (key) => key }}>
```

Comment:
the default value is not needed here

Reply:
If I don't set it the pieces of code depending on `t` set on the translation context fail.